packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/examples/getting-started/generate-extension.bash

```diff
@@ -1,2 +1,2 @@
 shopify upgrade
-shopify app generate extension
+POLARIS_UNIFIED=true shopify app generate extension
```

Comment:
Do you know what the plan is for this env var? In other words, should we be publicly documenting it as an option, or is it there for internal use for now and will be the default behavior after Oct 1?